webpack.config.js

```diff
@@ -27,12 +26,7 @@ module.exports = {
   mode: 'production',
   optimization: {
     ...commonConfig.optimization,
-    minimizer: [
-      new TerserJSPlugin({
-        /* Dexie has issues with UglifyJS */
-        exclude: /dexie/g,
-      }),
-    ],
+    minimize: true,
```

Comment:
this doesnt cause any issues?

Reply:
no, I tested it locally with production version, even with no sourceMap to be sure. tried sending message, deleting, saving backup, recovering from backup.